docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -24,52 +26,58 @@ In BODS, the following properties are used to represent such information disclos
-   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included in this primary Ownership-or-control Statement if known; and
-   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'.
-
-2. If beneficial ownership is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
+   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included if known; and
+   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'. If an interest is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
 
 Intermediaries
 ^^^^^^^^^^^^^^
 
-Where beneficial ownership is known to be exercised indirectly, via known intermediary entities, this SHOULD be represented in addition to the above. In particular:
+Where beneficial ownership is known to be exercised indirectly, via known intermediary elements, this SHOULD be represented in addition to the above. In particular:
 
-1. The chain of known intermediary entities SHOULD be represented by *secondary* Entity Statements, Ownership-or-control Statements and Person Statements.
-2. These secondary statements SHOULD link the beneficial owner’s Statement to entity X’s statement indirectly.
-3. These secondary statements SHOULD all have ``isComponent`` set to 'true'.
-4. These secondary statements SHOULD all have their ``statementID`` values listed in the ``componentStatementIDs`` array of the primary Ownership-or-control Statement.
-5. When the primary Ownership-or-control Statement is published in a BODS file:
+1. The chain of known intermediary elements SHOULD be represented by *secondary* records.
+2. These secondary records SHOULD link the beneficial owner’s record to entity X’s record indirectly.
+3. These secondary records SHOULD all have ``isComponent`` set to 'true'.
+4. These secondary records SHOULD all have their ``recordId`` values listed in the ``componentRecords`` array of the ``recordDetails`` for the primary relationship.
+5. When ``recordDetails`` for the primary relationship are published in a BODS file:
 
-   a. all secondary statements referenced from ``componentStatementIDs`` MUST also be published in that file;
-   b. all secondary statements must appear before the primary Ownership-or-control Statement in the list of statements.
+   a. Statements for all secondary records referenced from ``componentRecords`` MUST also be published in that file;
+   b. Statements for all secondary records must appear before the the first Statement for the primary relationship.
```

Comment:
```suggestion
   * statements for all secondary records must appear before the the first Statement for the primary relationship
```

Reply:
make sure however the primary relationship is referred to is consistent with the lead in line